Add meta descriptions to Lisk-SDK, closes #371 (#387)
LiskHQ/lisk-docs: modules/ROOT/pages/lisk-elements/packages/transactions.adoc

= Lisk Elements Transactions
Mona Bärenfänger <mona@lightcurve.io>

> :description: Technical reference for the Transactions package of Lisk Elements. It copmrises usage examples, available options and example responses.

Reviewer: ```suggestion
:description: Technical references regarding the Transactions packages of Lisk Elements. This consists of usage examples, available options and example responses.
```
